modules/pytket-quantinuum/docs/intro.txt

```diff
@@ -17,6 +17,104 @@ and Windows. To install, run:
+
+**Job Statuses**
+
+When using the Quantinuum ``Backend`` to run circuits there are several possible circuit statuses.
+
+* queued - The job has been queued but has not yet been run.
+* running - circuit is currently being run on the device/emulator.
+* completed - the job has finished.
+* failed - job has failed.
+* cancelling - the job is in the process of being cancelled.
+* cancelled - the job has been cancelled.
+
+**Web portal:**
+https://um.qapi.honeywell.com/user
+
+**Default Compilation**
+
+Every ``Backend`` in pytket has its own ``default_compilation_pass`` method. 
+This method applies a sequence of optimisations to a circuit depending on the value of an ``optimisation_level`` parameter. 
+This default compilation will ensure that the circuit meets all the constraints required to run on the ``Backend``. 
+The passes applied by different levels of optimisation are specified in the table below.
+
+
+.. list-table:: **Default compilation pass for the Quantinuum Backend**
+   :widths: 25 25 25
+   :header-rows: 1
+
+   * - optimisation_level = 0
+     - optimisation_level = 1 [1]
+     - optimisation_level = 2
+   * - `DecomposeBoxes <https://cqcl.github.io/tket/pytket/api/passes.html#pytket.passes.DecomposeBoxes>`_
+     - `DecomposeBoxes <https://cqcl.github.io/tket/pytket/api/passes.html#pytket.passes.DecomposeBoxes>`_
+     - `DecomposeBoxes <https://cqcl.github.io/tket/pytket/api/passes.html#pytket.passes.DecomposeBoxes>`_
+   * - self.rebase_pass [2]
+     - `SynthesiseTket <https://cqcl.github.io/tket/pytket/api/passes.html#pytket.passes.SynthesiseTket>`_
+     - `FullPeepholeOptimise <https://cqcl.github.io/tket/pytket/api/passes.html#pytket.passes.FullPeepholeOptimise>`_
+   * -
+     - self.rebase_pass [2]
+     - `SynthesiseTket <https://cqcl.github.io/tket/pytket/api/passes.html#pytket.passes.SynthesiseTket>`_
+   * -
+     - `auto_squash_pass [3] <https://cqcl.github.io/tket/pytket/api/passes.html#pytket.passes.auto_rebase.auto_squash_pass>`_
+     - self.rebase_pass [2]
+   * -
+     - `SimplifyInitial [4] <https://cqcl.github.io/tket/pytket/api/passes.html#pytket.passes.SimplifyInitial>`_
+     - `auto_squash_pass [3] <https://cqcl.github.io/tket/pytket/api/passes.html#pytket.passes.auto_rebase.auto_squash_pass>`_
+   * -
+     -
+     - `SimplifyInitial [4] <https://cqcl.github.io/tket/pytket/api/passes.html#pytket.passes.SimplifyInitial>`_
+
+* [1] If no value is specified then ``optimisation_level`` defaults to a value of 1.
+
+* [2] ``self.rebase_pass`` is a rebase that converts the circuit to the Quantinuum native gate set {Rz, PhasedX, ZZMax, ZZPhase}.
+
+* [3] ``auto_squash_pass`` has arguments ``auto_squash_pass({OpType.PhasedX,OpType.Rz})``
+
+* [4] ``SimplifyInitial`` has arguments  ``SimplifyInitial(allow_classical=False, create_all_qubits=True, xcirc=xcirc)``
```

Comment:
This is incomplete without code context (what is `xcirc`?). Do we wantthis level of detail here?